### ISSUE TITLE
GitHub starred counts only show for installed plugins

### ISSUE THREAD

**Issue by dwilcomb** [buzzfeed/Sketch-Toolbox#43]
Observed behavior:  All plugin counts were zero.  I had three plugins installed.  I logged into GitHub and starred a plugin, then refreshed my "installed" list.  The plugin count updated on refresh, but only for my installed plugins.  All other plugins in the "All" list show zero stars - even after refresh.
![screenshot](https://cloud.githubusercontent.com/assets/5281611/6333934/29dc2196-bb5f-11e4-88b7-67f424eff816.png)

**Comment by conan007ai** [comment]
My version remains 0.

**Comment by shahruz** [comment]
Fix coming!

**Comment by csi7585** [comment]
Looking at the code, it seems the call to get the data is simple.  But perhaps the issue is that in the current state, you would have to make an async request for every single on on the Sketch Plugins list.

**Comment by nolastan** [comment]
Correct, @csi7585. And since Github's [rate limit](https://developer.github.com/v3/#rate-limiting) is 60 requests per hour, we won't be able to get the stargazer count for every plugin.

Some ideas:
- Require Github authentication, allowing 5,000 requests per hour
- Create a web service for the app to communicate with to get the stargazer count
- Remove stars from the app directory

Any other ideas?

**Comment by csi7585** [comment]
I think the web service idea could work.  You could just have it get the count maybe once a day or even once a week, and cache it.
